lib/json_matchers/matcher.rb

```diff
@@ -10,16 +10,22 @@ def initialize(schema_path, **options)
     def matches?(response)
       @response = response
 
-      JSON::Validator.validate!(
-        schema_path.to_s,
-        response.body,
-        options,
-      )
-    rescue JSON::Schema::ValidationError => ex
-      @validation_failure_message = ex.message
-      false
-    rescue JSON::ParserError
-      raise InvalidSchemaError
+      begin
+        add_schemata_to_document_store
```

Comment:
https://github.com/valscion/json_matchers/commit/8cac06082b3474f8e4c4a38fd884ebeb773597e6

With this, the change I have to make is to add

```ruby
JsonMatchers.add_schemata_to_document_store
```

to the `spec_helper.rb` of my own project. I kind of like this approach — if you think that this would be a good approach, then I can whip up a proper PR for it.